client/src/pages/Social.jsx

```diff
@@ -166,10 +174,37 @@ class Social extends Component {
       this.getUserInfo();
     }
     this.getCheckInLocation();
+    console.log(this.props.match.params.checkInLocation)
```

Comment:
Funny, this is the exact code that I was gonna write! I'm glad that you were able to implement the switch case here.

Reply:
It seems to be the best option available for sure!